docs/configurations.md

```diff
@@ -0,0 +1,77 @@
+# Configuration Variables
+
+Configuration variables allow you to provide runtime configuration to the host OS and Supervisor. These variables all begin with `BALENA_` or the legacy prefix `RESIN_`. Beginning with `Supervisor v7.0.0`, a number of them automatically will be sent to the cloud on provision.
+
+__Note:__ Configuration variables defined in the dashboard will not apply to devices in local-mode.
+
+## Variable list
+
+This list contains configuration variables that can be used with all balena devices, some of which will automatically appear for devices with Supervisor v7.0.0 and greater. While they may not automatically populate in the dashboard, most of these variables can still be used with older Supervisor versions, so be sure to check the *Valid from* column:
+
+Name | Default | Description | Valid from
+--- | --- | --- | ---
+BALENA_SUPERVISOR_OVERRIDE_LOCK | false | Override any existing locks preventing application changes. | v1.0.0
+BALENA_SUPERVISOR_VPN_CONTROL | true | Enable/ Disable VPN service on device. | v1.1.0
+BALENA_SUPERVISOR_CONNECTIVITY_CHECK | true | Enable / Disable VPN connectivity check | v1.3.0
+BALENA_SUPERVISOR_LOG_CONTROL | true | Enable / Disable logs being sent to the balena API | v1.3.0
+BALENA_SUPERVISOR_POLL_INTERVAL | 900000 | Define the balena API poll interval in milliseconds. This interval will only matter if the device is not connected to the VPN at the time an update is pushed, or if BALENA_SUPERVISOR_INSTANT_UPDATE_TRIGGER is set to false. Starting from Supervisor v9.13.0, the Supervisor will use a random time between 0.5 and 1.5 times this poll interval each time it checks the balenaCloud API. The minimum value for this variable is defined by the balenaCloud backend, and may vary. | v1.3.0
+BALENA_SUPERVISOR_DELTA_REQUEST_TIMEOUT | 30000 | Millisecond time within which an HTTP response containing delta metadata must be returned by the delta server. | v2.9.0
+BALENA_SUPERVISOR_LOCAL_MODE | false | Enable / Disable local-mode. | v4.0.0
```

Comment:
@20k-ultra I'll summarize the context surroundings discussions of some of these config vars from the FD thread:
`*_SUPERVISOR_OVERRIDE_LOCK` and `*_SUPERVISOR_LOCAL_MODE` are reported through the SDK and on the dashboard as either `"0"` or `"1"`. The format is a `string int`. There might be other vars but these are the main ones that I saw had the difference. However, these vars are documented here as being `true` or `false`. While the Supervisor has the function `checkTruthy` to convert `"0"` to `false` and so on, this documentation still differs from the dashboard / SDK, and both are user-facing resources. I'm wondering if it would be beneficial, at least, to note somewhere in this PR that `false` here might be listed as `"0"` on the SDK/dashboard?

Reply:
@20k-ultra yo! I know you're busy so if you tell me your decision on this, I can take this off your plate :D

Reply:
I understand what you're saying but don't think it's necessary. The point is that the documentation just has to convey what the default boolean value is. If we add 1 or 0 then we'll just have to keep changing our documentation to match what values we accept which as of right now are [10](https://github.com/balena-os/balena-supervisor/blob/14efec2c7df29107a7a24f2080d4868abf6ca528/src/lib/validation.ts#L16-L17) different values.

Reply:
I modified the tables to say what the data type is for each value. This was needed because for some configs I had undefined for the default but it wasn't clear what they should be set to.

Reply:
Oh geez. The 10 different values thing is a bit spooky, I feel like limiting allowed values to a simple true/false boolean would have been better. I see your point though.